packages/react-spinbutton/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/node"],
```

Comment:
This shouldn't be `node`, it should probably be `react` (and then disable the naming-convention rule for any compat interface names which already start with I)